QUEST_LV_0100.tsv

```diff
@@ -7554,7 +7554,7 @@ QUEST_LV_0100_20150714_007553	$Hand them over to Guard Molly
 QUEST_LV_0100_20150714_007554	$You've obtained all the materials that are needed to make Stringers. Hand them over to Guard Molly.
 QUEST_LV_0100_20150714_007555	$You've obtained all the materials that are needed to make Stringers. Ask Guard Molly about what to do next.
 QUEST_LV_0100_20150714_007556	$Set the Stringers
-QUEST_LV_0100_20150714_007557	$To prevent Pantos from straying to other regions, please set the Stringers in Srautas Canyon and Nefritas Cliffs.
+QUEST_LV_0100_20150714_007557	$To prevent Pantos from straying to other regions, please set the Stringers in Srautas Gorge and Nefritas Cliffs.
```

Comment:
I came across this line while changing instances of 'Srautas Canyon', any clue where this quest happens?  I dunno what 'Stringers' are, and would like to do the quest to get some context.